change/@fluentui-react-carousel-b0491798-cc10-487f-a3c0-98bcd6016f7f.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33401&commitId=687139dab19dfa1929f2f880f973abd016532971&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|7| Changed|
|Avatar Converged.Badge Mask RTL.chromium.png|5| Changed|

</details>


  <details><summary><b>Drawer</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.overlay drawer full.chromium.png|2898| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>